### ISSUE TITLE
Create Xamarin.Forms samples for iOS, Android, and UWP

### ISSUE THREAD

**Issue by JimSEOW** [gmlwns2000/TensorFlowSharp#1]
It would be great for the proof of concept.
You may be interested of this .[Net Deep Learning discussion
](https://github.com/Microsoft/CNTK/issues/960#issuecomment-306508050)

If you have idea how to show simple example of CNTK c# for Xamarin, it will be really exciting for the community

**Comment by gmlwns2000** [comment]
I agree to make some samples on the project, but these days I am busy because of school. :(
And thank you for letting me know about CNTK. It seems exciting

**Comment by JimSEOW** [comment]
@gmlwns2000 plsvtake your time. Thx for trying. Pls spread CNTK too. We try to get more people to support .Net deep learning for client and front end. There is also gitter for cntk. I support both tensorfliwsharp and Cntk for .NET

**Comment by JimSEOW** [comment]
@gmlwns2000 which share your vision of .NET Deep learning to Daniel of Microsoft  [here](https://github.com/Microsoft/CNTK/issues/960#issuecomment-315049580)

**Comment by gmlwns2000** [comment]
@JimSEOW  Thank you for sharing it.
